FiveCalls/FiveCalls/Localizable.strings

```diff
@@ -47,3 +47,5 @@
 "scheduled-reminders-description"     = "Turn these on to get a quick local reminder to make your 5 calls.";
 "line-busy"                           = "Line Busy? ▼";
 "choose-a-number"                     = "Select a Different Office";
+"think-before-copying-alert-title"    = "Please Consider Before Texting, Faxing, or Emailing";
```

Comment:
I think the meaning is more like "Think about", rather than a specific option. I figure, we're in an app called "5 Calls". Any consideration would kinda have to take Calling into account.